### PR TITLE
feat: update model-service related i18n

### PR DESCRIPTION
**Pull request by Sujin-Kim1**
follows #1801

### CODE REVIEW

File: resources/i18n/ko.json [hunk 1]
```diff
@@ -382,7 +382,32 @@
...
+    "DesiredSessionCount": "원하는 세션 수",
+    "EndpointName": "엔드포인트 이름",
+    "EndpointId": "엔드포인트 ID",
+    "Controls": "제어",
+    "Status": "상태",
+    "RoutingsCount": "라우팅 수",
+    "Active/Total": "활성/전체",
+    "Public": "공개",
+    "EditModelService": "모델 서비스 수정",
+    "OnlyAllowsNonNegativeIntegers": "양의 정수만 사용 가능합니다.",
```
Comment (yomybaby) [inline resources/i18n/ko.json]:
```suggestion
    "OnlyAllowsNonNegativeIntegers": "0 이상을 입력해주세요.",
```

File: resources/i18n/ko.json [hunk 2]
```diff
@@ -382,7 +382,32 @@
     "Running": "실행중",
     "Finished": "종료",
     "Others": "기타",
-    "Services": "모델 서비스"
+    "Services": "모델 서비스",
+    "StartNewServing": "새 서비스 시작",
+    "ServiceName": "서비스 이름",
+    "OpenToPublic": "앱을 외부에 공개",
```
Comment (yomybaby) [inline resources/i18n/ko.json:388]:
```suggestion
    "OpenToPublic": "공개",
```

Comment (agatha197) [reply resources/i18n/ko.json:388]: @yomybaby I just followed [this line](https://github.com/lablup/backend.ai-webui/blob/6b0e4513c681781eb8e849fc2825efaa0e381539/resources/i18n/ko.json#L282). Is it better to change?

Comment (yomybaby) [reply resources/i18n/ko.json:388]: Okay, I'll merge without this suggestion now and make changes later if necessary. 👍

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**